site/app/controllers/grading/ImagesController.php

```diff
@@ -126,7 +101,10 @@ public function ajaxUploadImagesFiles() {
         }
 
         if (isset($uploaded_files[1])) {
+            $users = $this->core->getQueries()->getListOfCourseUsers();
+            // For each item that was uploaded
```

Comment:
Can remove this comment and the others added in this function. The code itself is readable enough on what it's doing, and these comments do not provide any further meaningful context.

Reply:
I worked with this code quite a bit and I didn't find it readable, thus the comments.  I stand behind keeping them.

Reply:
If the code is hard to understand / read that these short one line comments are necessary, that's a code smell and that this should be cleaned / modified in some way to be clearer. However, I'm not convinced (yet) and would like to perform a dive on why you think the comment is necessary so can think on making this block of code more readable.

Let's start with this comment: `// Item was a zip file`

The line of code it comments: `if ($uploaded_files[1]["is_zip"][$j] === true) {`.

What does this comment add to understanding this line is testing that something is a zip?

Reply:
I don't have a problem with this comment, let's not make a fuss over it